python/ray/data/tests/test_dataset_formats.py

```diff
@@ -2998,6 +2998,18 @@ def test_csv_read_with_column_type_specified(shutdown_only, tmp_path):
     assert ds.to_pandas().equals(expected_df)
 
 
+def test_csv_read_filter_no_file(shutdown_only, tmp_path):
+    df = pd.DataFrame({"one": [1, 2, 3], "two": ["a", "b", "c"]})
+    table = pa.Table.from_pandas(df)
+    path = os.path.join(str(tmp_path), "test.parquet")
+    pq.write_table(table, path)
+
+    error_message = "Not found any input file to read from"
+    with pytest.raises(ValueError) as e:
+        ray.data.read_csv(path)
+    assert error_message in str(e.value)
```

Comment:
Nit: `pytest` natively supports pattern matching on exceptions! https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions

```suggestion
    error_message = "Not found any input file to read from"
    with pytest.raises(ValueError, match=error_message):
        ray.data.read_csv(path)
```

Reply:
@clarkzinzow - nice, updated.